grammar.js

```diff
@@ -1,6 +1,8 @@
 /*
 * Known bugs:
 * Placing a link description at the end of an unordered list element breaks the generic_list
+* m_Whitespace is not preserved properly and does not fully function, causing errors
+* Forgot to add support for ` inline code blocks
```

Comment:
Is this still valid?

Reply:
This isn't applicable anymore since `m_Whitespace` isn't used anymore yeah. All of the other changes that you showed are valid and are not the results of an invalid merge thankfully :)

It seems the problems stems directly from the way attached modifiers and whitespace is handled and so we'd have to look there.

Reply:
I see thanks! Here I was actually asking about the missing support for inline code but good that the other thing is also resolved :smile: